qstack/spahm/rho/bond.py

```diff
@@ -58,7 +58,7 @@ def bond(mols, dms,
     for imol, (mol, dm) in enumerate(zip(mols,dms)):
         if printlevel>0: print('mol', imol, flush=True)
         for iomod, omod in enumerate(omods):
-            DM  = utils.dm_open_mod(dm, omod) if spin else dm
```

Comment:
AAAh I think I got it now!
I think the `spin` argument is not supposed to hold the spins of the molecules but rather decide whether to treat to system as open-shell or not!
Rings a bell?

Reply:
I am confused now because looks like inside of `bond()` it is always supposed to be a list

Reply:
but also it's not needed for anything so should be changed to a bool is_openshell

Reply:
which would duplicate `omods`

Reply:
so I think we don't pass `spin` to bond() anymore and check is `omods` should be `[None]` already in `get_repr()`. what do you think?